mxpy-up.py

```diff
@@ -16,38 +17,34 @@
 
 def main():
     parser = ArgumentParser()
-    parser.add_argument("--modify-path", dest="modify_path", action="store_true", help="whether to modify $PATH (in profile file)")
-    parser.add_argument("--no-modify-path", dest="modify_path", action="store_false", help="whether to modify $PATH (in profile file)")
+    parser.add_argument("--modify-path", dest="modify_path", action="store_true", help="(deprecated, not used)")
```

Comment:
Kept for one more release, to ensure a smooth transition to mxpy v6 in IDE, as well - the VSCode extension was setting `--no-modify-path`.

Reply:
Not kept anymore. We'll prepare a hotfix for the IDE when mxpy 6 is released.